### PR TITLE
[front] - fix(navigation): sort items by name

### PR DESCRIPTION
**Pull request by JulesBelveze**
## Description

This PR aims at sorting the elements within the sub menu

## Risk

None

## Deploy Plan

Deploy `front`

### CODE REVIEW

File: front/components/vaults/VaultSideBarMenu.tsx [hunk 1]
```diff
@@ -511,7 +516,7 @@ const VaultDataSourceViewSubMenu = ({
     >
       {isExpanded && (
         <Tree isLoading={isVaultDataSourceViewsLoading}>
-          {vaultDataSourceViews.map((ds) => (
+          {sortedViews.map((ds) => (
```
Comment (Fraggle) [inline front/components/vaults/VaultSideBarMenu.tsx:519]: We usually sort by "importance". (there is a function for that)

**Review by Fraggle — APPROVED** [review]

**Review by tdraier — APPROVED** [review]

File: front/components/vaults/VaultSideBarMenu.tsx (hunk 1 above)
Comment (JulesBelveze) [reply front/components/vaults/VaultSideBarMenu.tsx:519]: @Fraggle what's the name of that function? 🤔

Comment (Fraggle) [reply front/components/vaults/VaultSideBarMenu.tsx:519]: orderDatasourceByImportance and co.

Comment (JulesBelveze) [reply front/components/vaults/VaultSideBarMenu.tsx:519]: Looked up and not really useful in the case of websites and folders as it doesn't sort alphabetically

**Pull request merged**